autopengui.py

```diff
@@ -51,16 +57,106 @@
 sdr = ('gnu radio', 'gqrx')
 mis = ('katoolin')
 all_tools = can + (b_w,) + (sdr,) + (mis,)
-
 tools_to_install = []
 broken = []
+chosen = ''
+cars = []
+connection = 1
+def set_chosen(input):
+	global chosen
+	chosen = input
+
+def get_chosen():
+	return chosen
+
+try:
+    db = MySQLdb.connect(host="autopen.c5eqmploekdx.us-east-2.rds.amazonaws.com", user="SHIFT", passwd="%&12SHilaE7", db="autopen", port=3306)
+except MySQLdb.Error as mysql_error:
+    print 'Cannot connect to database'
+    print mysql_error
+    connection = 0
+else:
+    cursor = db.cursor();
+    cursor.execute("SELECT * FROM Cars");
+    numrows = cursor.rowcount
+    cars = cursor.fetchall()
+    cursor.execute("SELECT * FROM Exploits");
+    exploits = cursor.fetchall()
+    db.close()
+    x = .1
+    y = .9
+    for row in cars:
+	#x = x + z
+	myString = "<CarsPage>:\n\tButton:\n\t\tid:"+str(row[1])+"\n\t\ttext: '" + str(row[1]) + "'\n\t\tsize_hint: .25, .1\n\t\tpos_hint: {'x':" + str(x) + ", 'y':" + str(y)+"}"
+	parttwo = "\n\t\ton_press:\n\t\t\troot.manager.transition.direction='left'\n"
+	partthree= "\t\t\troot.manager.transition.duration=.5\n\t\t\troot.manager.current='carinfo'"
+	y -= .1
+        partfour = "\n\t\ton_press:\n\t\t\troot.manager.get_screen('carinfo').work('"+ str(row[1])+"')"
+	#parttwo = "\n\t\ton_press:\n\t\t\troot.cars("+str(row[0])+")"
+	Builder.load_string(myString+parttwo+partthree+partfour)
+	myString = ''
+class CustomDropDown(DropDown):
+    pass
 
+class CarInfoPage(Screen):
+        label_wid = ObjectProperty()
+        exploitbox_wid = ObjectProperty()
+        def work(self,stuff):
+		if connection == 0:
+			print "Cannot connect to Database"
+			return
+                self.label_wid.text = stuff
+                for i in cars:
+                        if i[1] == stuff:
+                                for t in exploits:
+                                        if t[1] == i[0]:
+                                                finalmessage = self.exploitbox_wid.text+"\n\n"+"Date Found: "+str(t[2])+"\n"+"Car Year: " + str(t[4])+"\n"+t[3]
+                                                self.exploitbox_wid.text = finalmessage		
+	pass
+	
 class ScrollableLabel(ScrollView):
     text = StringProperty('')
 
 class WelcomePage(Screen):
     pass
 
+class DataEntryPage(Screen):
+    mandate_wid = ObjectProperty()
+    carmodel_wid = ObjectProperty()
+    exploit_wid = ObjectProperty()
+    exploitdate_wid = ObjectProperty()
+    def do_action(self):
+        if connection == 0:
+		print 'No database connection'
+		return
+	model = self.carmodel_wid.text
+	year = self.mandate_wid.text
+	print (self.exploitdate_wid.text)
+	print (self.exploit_wid.text)
+        db = MySQLdb.connect(host="autopen.c5eqmploekdx.us-east-2.rds.amazonaws.com", user="SHIFT", passwd="%&12SHilaE7", db="autopen", port=3306)
```

Comment:
I don't think that including the username and password of our server is a good idea here. This would give anyone access to the database.